R/tspa_plot_revised.R

```diff
@@ -0,0 +1,123 @@
+tspa_plot <- function(tspa_fit,
```

Comment:
Do you intend to have this "revised" file to replace the original file? Generally it is not a good idea to keep two versions of the same file using Git, as that means different file path history. Just keep one copy; when you commit a file, the history is already saved, so you don't need to have a separate file.

Reply:
Sounds good! I deleted the old version and renamed this file to "tspa_plot.R"